lib/matplotlib/contour.py

```diff
@@ -614,8 +618,9 @@ def labels(self, inline, inline_spacing):
             trans = self.get_transform()
             label_width = self._get_nth_label_width(idx)
             additions = []
-            for subpath in self._paths[icon]._iter_connected_components():
-                screen_xys = trans.transform(subpath.vertices)
+            for subpath in trans.transform_path(
+                    self._paths[icon])._iter_connected_components():
+                screen_xys = subpath.vertices
```

Comment:
I think you could save the inverse transform below by leaving the for loop as it was, and then calculating `tsubpath = trans.transform_path(subpath)` inside the loop and then use subpath/tsubpath in the right locations below.